### PR TITLE
feat(sample): view for sample

### PR DESCRIPTION
**Pull request by Daryl-L**

### CODE REVIEW

File: packages/samples/mvp-dapp/src/views/tx.view.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { CellChangeData } from '@ckb-js/kuai-models'
+import { Cell, helpers, commons } from '@ckb-lumos/lumos'
+
+export class Tx {
+  static async toJson(
+    txSkeleton: helpers.TransactionSkeletonType,
```
Comment (Keith-CY) [inline packages/samples/mvp-dapp/src/views/tx.view.ts:6]: Is the `skeleton` available for unlocking the omnilock?

File: packages/samples/mvp-dapp/src/views/read.view.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+// import type { ItemData } from '../record/record.model'
+
```
Comment (Keith-CY) [inline packages/samples/mvp-dapp/src/views/read.view.ts:2]:
```suggestion
```

File: packages/samples/mvp-dapp/src/views/load.view.ts [hunk 3]
```diff
@@ -0,0 +1,60 @@
+// import type { StoreType } from '../record/record.model'
+
```
Comment (Keith-CY) [inline packages/samples/mvp-dapp/src/views/load.view.ts:2]:
```suggestion
```

File: packages/samples/mvp-dapp/src/views/load.view.ts [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    return JSON.stringify({
+      profile,
+      addresses,
+      custom,
+      dweb,
+    })
+  }
+}
+
+interface Item {
```
Comment (Keith-CY) [inline packages/samples/mvp-dapp/src/views/load.view.ts:57]: Declaration `interface Item` could be placed with `type ItemData` and `type StoreType`

File: packages/samples/mvp-dapp/src/views/load.view.ts [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+  data: {
+    profile: ItemData[]
+    addresses: ItemData[]
+    custom: ItemData[]
+    dweb: ItemData[]
+  }
+}
+
+export class Load {
+  static toJson(storage: StoreType): string {
```
Comment (Keith-CY) [inline packages/samples/mvp-dapp/src/views/load.view.ts:19]: The method name indicates a `JSON` is returned, but a string is returned in fact.

File: packages/samples/mvp-dapp/src/views/read.view.ts (hunk 2 above)
Comment (Daryl-L) [reply packages/samples/mvp-dapp/src/views/read.view.ts:2]: Will be uncommented after model branch merged.

File: packages/samples/mvp-dapp/src/views/load.view.ts (hunk 3 above)
Comment (Daryl-L) [reply packages/samples/mvp-dapp/src/views/load.view.ts:2]: Will be uncommented after model branch merged.

File: packages/samples/mvp-dapp/src/views/read.view.ts (hunk 2 above)
Comment (Keith-CY) [reply packages/samples/mvp-dapp/src/views/read.view.ts:2]: Gotcha

File: packages/samples/mvp-dapp/src/views/load.view.ts (hunk 4 above)
Comment (Daryl-L) [reply packages/samples/mvp-dapp/src/views/load.view.ts:57]: Just refer to this https://github.com/Magickbase/kuai-mvp-dapp-ui/blob/develop/utils/mock.ts

Comment (Keith-CY) [reply packages/samples/mvp-dapp/src/views/load.view.ts:57]: The point is that `type declaration` could be placed together, and should be placed before where it's used.

**Comment by yanguoyu** [comment]
I think `ctx.ok`'s first parameter can be an object, why do we need to transfer it to a string?

**Comment by Daryl-L** [comment]
> I think `ctx.ok`'s first parameter can be an object, why do we need to transfer it to a string?

I just referred to what @PainterPuppets did.

File: packages/samples/mvp-dapp/src/views/tx.view.ts (hunk 1 above)
Comment (Keith-CY) [reply packages/samples/mvp-dapp/src/views/tx.view.ts:6]: Any update on this question?

Comment (Daryl-L) [reply packages/samples/mvp-dapp/src/views/tx.view.ts:9]: Supported, in https://github.com/ckb-js/kuai/pull/127/commits/cdb1aad571e0b66fe51ebd850896d4b5016b3a97.

**Review by yanguoyu — APPROVED** [review]

File: packages/samples/mvp-dapp/src/views/tx.view.ts [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+    txSkeleton.update('outputs', (outputs) => {
+      return outputs.push(...outputCells)
+    })
+
+    for (const [input, witness] of inputData) {
+      txSkeleton.update('inputs', (inputs) => inputs.push(input))
+
+      txSkeleton.update('witnesses', (witnesses) => {
+        if (witness === '0x' || witness === '') {
+          const omniLock = config.getConfig().SCRIPTS.OMNI_LOCK as NonNullable<config.ScriptConfig>
```
Comment (Keith-CY) [inline packages/samples/mvp-dapp/src/views/tx.view.ts:22]: Seems that chain type `mainnet/testnet` is not set, but we can do it later.

**Review by Keith-CY — APPROVED** [review]

**Pull request merged**